elastic4s-core/src/main/scala/com/sksamuel/elastic4s/Reader.scala

```diff
@@ -7,4 +12,94 @@ trait Reader[-U] {
 
 trait HitAs[T] {
   def as(hit: RichSearchHit): T
-}
\ No newline at end of file
+}
+
+trait HitRead[T] {
+  def as(hit: RichSearchHitLike): T Or Every[ErrorMessage]
+}
+
+trait HitFieldRead[T] {
+  def as(prefix:String)(hit: RichSearchHitField): T Or Every[ErrorMessage]
+  def as(hit: RichSearchHitField): T Or Every[ErrorMessage]=as("")(hit)
+}
+
+trait LPHitFieldReads{
+  implicit def hitRead2HitFieldRead[T](implicit hitRead:HitRead[T]) =new HitFieldRead[T] {
+    import scala.collection.JavaConverters._
+    override def as(prefix: String)(hit: RichSearchHitField): Or[T, Every[ErrorMessage]] = hit match {
```

Comment:
All other methods in the PR use `T Or Every[ErrorMessage]`. Just this method seems to be inconsistent.